lib/cli/src/commands/app/purge_cache.rs

```diff
@@ -0,0 +1,54 @@
+//! Get information about an edge app.
+
+use super::util::AppIdentOpts;
+use crate::{
+    commands::AsyncCliCommand,
+    opts::{ApiOpts, ItemFormatOpts},
+};
+
+/// Purge caches for applications.
+///
+/// Cache scopes that can be cleared:
+/// * InstaBoot startup snapshots
+///   Will delete all existing snapshots.
+///   New snapshots will be created automatically.
+#[derive(clap::Parser, Debug)]
+pub struct CmdAppPurgeCache {
+    #[clap(flatten)]
+    #[allow(missing_docs)]
+    pub api: ApiOpts,
+    #[clap(flatten)]
+    #[allow(missing_docs)]
+    pub fmt: ItemFormatOpts,
+
+    #[clap(flatten)]
+    #[allow(missing_docs)]
+    pub ident: AppIdentOpts,
+}
+
+#[async_trait::async_trait]
+impl AsyncCliCommand for CmdAppPurgeCache {
+    type Output = ();
+
+    async fn run_async(self) -> Result<(), anyhow::Error> {
+        let client = self.api.client()?;
+        let (_ident, app) = self.ident.load_app(&client).await?;
+
+        let version_id = app.active_version.id;
+
+        let name = format!("{}/{}", app.owner.global_name, app.name);
```

Comment:
nit: use `app.global_name` instead?

Reply:
sidenote: all resources have a `globalName` field on the backend that should be used when refering to it, instead of building the name manually everywhere. This is to keep things consistent across clients/commands.

Reply:
it will be in 15 minutes ;)

Reply:
If you don't want to fix this by using globalName, then please use another nomenclature for the app.

We use:
```suggestion
        let name = format!("{} ({})", app.name, app.owner.global_name);
```